spec/rubocop/cop/style/method_name_spec.rb

```diff
@@ -49,6 +49,16 @@ def +(other)
       END
     end
 
+    it 'accepts unary operator definitions' do
+      expect_no_offenses(<<-END.strip_indent)
```

Comment:
Sublime and atom by default

![image](https://cloud.githubusercontent.com/assets/2085622/26080885/57e788b6-397d-11e7-946d-24b3225e96b6.png)

![image](https://cloud.githubusercontent.com/assets/2085622/26080893/6000f582-397d-11e7-866e-b1838c2a00c7.png)

looks like there is an extension for vim too https://github.com/joker1007/vim-ruby-heredoc-syntax